src/Runner/MediaInfoCommandRunner.php

```diff
@@ -57,21 +54,15 @@ public function __construct(
         ];
         $finalCommand = [$this->command];
 
-        $i = 0;
         foreach ($args as $value) {
-            $var = 'MEDIAINFO_VAR_'.$i++;
-            $finalCommand[] = '$'.$var;
-            $env[$var] = $value;
+            $finalCommand[] = $value;
         }
 
-        $finalCommandString = implode(' ', $finalCommand);
-
         if (null !== $process) {
-            $process->setCommandLine($finalCommandString);
             $process->setEnv($env);
             $this->process = $process;
         } else {
-            $this->process = new Process($finalCommandString, null, $env);
+            $this->process = new Process($finalCommand, null, $env);
```

Comment:
With symfony/process:~2.7 [Process constructor](https://github.com/symfony/symfony/blob/2.7/src/Symfony/Component/Process/Process.php#L128) doesn't support array as first argument

Reply:
Thanks, seems you're right. It looks like that array type argument has been added within 3.3 so if you want to keep Symfony release support then this merge request has to wait till Nov this year. 

ref. https://symfony.com/doc/current/contributing/community/releases.html

Anyways, current way of passing arguments - via ENV variables - seems a bit ridiculous. Isn't it?

Reply:
I'm ok, to stop using vars as way of passing arguments.

Reply:
@ireneuszsawicki Could you make the changes ?

Reply:
I'm sorry @mhor but I'm afraid I would not be able to provide some cross-version compatible solution in reasonable time (below few weeks). I'm terribly busy recently.

Reply:
I stumbled upon a deprecation warining in Symfony caused by this. 
Since i couldn't really find a quick and solid way to check version in code, could the following 'hack' not do the trick untill Symfony has officially ended the 2.x support (ends Q4 2018, security fixes Q4 2019).

Try calling `setCommandLine()` with an array as argument, wrapped in a try/catch block, if exception is cought fall back to string argument?

Below the deprecation warning Symfony is printing (on 4.2.x)
```
[2018-12-21 10:14:06] php.INFO: User Deprecated: Passing a command as string when creating a "Symfony\Component\Process\Process" instance is deprecated since Symfony 4.2, pass it as an array of its arguments instead, or use the "Process::fromShellCommandline()" constructor if you need features provided by the shell. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Passing a command as string when creating a \"Symfony\\Component\\Process\\Process\" instance is deprecated since Symfony 4.2, pass it as an array of its arguments instead, or use the \"Process::fromShellCommandline()\" constructor if you need features provided by the shell. at /var/www/app/vendor/symfony/process/Process.php:147)"} []
```